reconcile/openshift_acme.py

```diff
@@ -151,8 +151,6 @@ def run(dry_run=False, thread_pool_size=10, internal=None,
         ob.realize_data(dry_run, oc_map, ri)
 
     except Exception as e:
-        msg = 'There was problem running openshift acme reconcile.'
-        msg += ' Exception: {}'
-        msg = msg.format(str(e))
-        logging.error(msg)
+        logging.exception('Exception running openshift acme reconcile. ' +
+                          '{}'.format(str(e)))
```

Comment:
No need to include the `str(e)`. The exception message will be in the traceback.

Reply:
logging.exception needs a message

Reply:
Yes it does. The redundant part is only the `str(e)`.

Reply:
PR updated